lib/server.js

```diff
@@ -189,3 +203,21 @@ module.exports = {
   }
 };
 
+function cleanUpAndExit() {
+  if (electronPids) {
```

Comment:
The final clean up will only be applicable when you change and save your code too fast which opens up a lot of electron apps.